### PR TITLE
MAUI-2491 : Prepare UG document for sublayer in Maui Maps control

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @vigneshrameshsync ,
 I have prepared UG content for sublayer in Maui maps control. Could you please review it and let me know if you have any concerns?

Regards,
Shalini.

### CODE REVIEW

File: MAUI/Maps/vector-layers/polygon-layer.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+title: Adding polygons in Maps control | Syncfusion
+description: Learn here all about adding the Polygons feature of Syncfusion .NET MAUI Maps (SfMaps) control to customize their appearances and more.
+platform: MAUI
+control: SfMaps
+documentation: ug
+---
+
+# Polygons in .NET MAUI Maps (SfMaps)
+
+Polygon layer is a sublayer that renders a group of `MapPolygon` on `MapShapeLayer`. This section helps to learn about how to add the polygons and customize them.
```
Comment (florenceliliana) [inline MAUI/Maps/vector-layers/polygon-layer.md:12]: Polygon layer is a sublayer that renders a group of `MapPolygon` on the `MapShapeLayer.` This section helps to learn about how to add the polygons and customize them.

File: MAUI/Maps/vector-layers/polygon-layer.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+## Adding polygons
+
+The `Polygons` is a collection of `MapPolygon`. Every single `MapPolygon` connects multiple coordinates through a `Points` property.
```
Comment (florenceliliana) [inline MAUI/Maps/vector-layers/polygon-layer.md:16]: The `Polygons` is a collection of `MapPolygon.` Every single `MapPolygon` connects multiple coordinates through a `Points` property.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/Maps/vector-layers/polygon-layer.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/polygon-layer.md]: Changed the content as mention above

File: MAUI/Maps/vector-layers/polygon-layer.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/polygon-layer.md:16]: Mentioned content is same as a UG default content.

**Comment by Shalini-Ashokan** [comment]
rebuild

**Review by vigneshrameshsync — APPROVED** [review]
The content is fine.

**Pull request merged**